### DISCUSSION TITLE
Funcionalidade na qual os participantes do RS/XP possam se encontrar no LinkedIn, com uma interface semelhante à tela de Speakers

### DISCUSSION THREAD

**Discussion by Giuzntt** [diego3g/rsxp-2023#37] [Ideas]
A tela de funcionalidade para que os participantes do RS/XP se encontrem no LinkedIn é uma ferramenta importante para promover o networking entre os participantes do evento. Como se trata de um evento voltado para desenvolvedores, é essencial criar oportunidades para que eles possam compartilhar conhecimento e fazer conexões relevantes para suas carreiras.

Ao conectar-se no LinkedIn com outros participantes do evento, os desenvolvedores podem ter acesso a novas oportunidades de emprego, projetos e até mesmo potenciais clientes. Além disso, essa conexão pode levar a novas amizades e parcerias profissionais, aumentando ainda mais a rede de contatos de cada participante.

Através da tela de funcionalidade de networking, é possível que os participantes do RS/XP encontrem pessoas com interesses em comum e experiências similares, facilitando assim a troca de informações e ideias. Essa interação pode ser enriquecedora para todos os envolvidos, permitindo que os desenvolvedores aprendam uns com os outros e se inspirem a buscar novos desafios em suas carreiras.

Em resumo, a tela de funcionalidade para networking no LinkedIn é uma ferramenta essencial para os participantes do RS/XP, pois permite que eles construam conexões valiosas e expandam suas redes de contatos profissionais. Com isso, os desenvolvedores podem se beneficiar tanto em termos de oportunidades profissionais quanto de crescimento pessoal e aprendizado.

**Comment by Mikkaiser** [comment]
Mas assim, como que seria essa parte de conexões pelo Linkedin? Seria tipo uma tela que permitisse você buscar o Linkedin de outra pessoa através da leitura de um QR Code da credencial ou algo assim?

Ou seria algo tipo: uma tela com todos os participantes listados, e você pode pesquisar a pessoa que você quer por lá, e assim ter as infos dela?

**Reply by Giuzntt** [reply]
A ideia de exibir outros participantes listados com LinkedIn, acredito que isso pode contribuir com ambiente de networking

**Reply by rpc1910** [reply]
Acho que essa funcionalidade poderia servir como um cartão de visita virtual, onde ao ler o QR Code da credencial seria exibido o perfil do usuário dentro do app com links para Liknedin, github, etc, facilitando o networking entre os participantes.

**Reply by leoferreiralima** [reply]
Gosto da ideia do cartão virtual, e ter o links afiliados como um linktree

**Comment by Giuzntt** [comment]
@diego3g o que acha?
